music_metadata_extractor/songdata.py

```diff
@@ -18,7 +18,11 @@ def __init__(self, input_url: str):
             logging.warning(e, stack_info=True)
             self.track: Track = Track(provider_id=None, name=provider_input.song_name, is_cover=None, original_id=None, popularity=None, year=None, explicit=None, image_id=None, genre=[])
             self.artists: List[Artist] = []
+        
+        self.yt_views = provider_input.song_ytviews
+        self.yt_date = provider_input.song_ytdate
+
 
 
     def __repr__(self):
-        return "<SongData(\n\ttrack=%s,\n\tartists=%s\n)>" % (self.track, self.artists)
+        return "<SongData(\n\ttrack=%s,\n\tartists=%s,\n\tviews=%s,\n\tdate=%s\n)>" % (self.track, self.artists , str(self.yt_views), str(self.yt_date))
```

Comment:
Continuing from previous comment, this would then just be: `self.extraAttrs: dict = extraAttrs`.

Reply:
Sure